pkg/terraformer/state.go

```diff
@@ -40,8 +40,8 @@ type terraformStateV4 struct {
 	Outputs map[string]outputState `json:"outputs"`
 }
 
-// getState returns the Terraform state as byte slice.
-func (t *terraformer) getState() ([]byte, error) {
+// GetState returns the Terraform state as byte slice.
+func (t *terraformer) GetState() ([]byte, error) {
```

Comment:
I guess you need to add the exported funcs to the Terraformer interface to be able to consume them from the provider-extensions, right?

https://github.com/gardener/gardener-extensions/blob/5f3a64cc8bf2cadac3a8ea6e09dddfaa474bb8c8/pkg/terraformer/types.go#L90

Also can you outline how would like consumption from the provider-extensions?

Reply:
Indeed, nice catch, thank you.